Tests/UnitTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -700,10 +698,10 @@ class PurchasesSubscriberAttributesTests: TestCase {
 
         expect(self.mockBackend.invokedPostReceiptData).to(beTrue())
         expect(self.mockStoreKit1Wrapper.finishCalled).toEventually(beTrue())
-        expect(self.mockSubscriberAttributesManager.invokedMarkAttributes) == true
-        expect(self.mockSubscriberAttributesManager.invokedMarkAttributesParameters!.syncedAttributes) == mockAttributes
-        expect(self.mockSubscriberAttributesManager.invokedMarkAttributesParameters!.appUserID) ==
-        mockIdentityManager.currentAppUserID
+        expect(self.mockSubscriberAttributesManager.invokedMarkAttributes).toEventually(beTrue())
```

Comment:
This is the only semantic change. Because of this refactor, attributes are marked as synced after the transactions are finished. The test below (like 732) already had `toEventually`. This is purely an ordering change and shouldn't have any impact.